docs/application/dotnet/guides/user-interface/nui/relative-layout.md

```diff
@@ -73,14 +98,525 @@ The `HorizontalAlignment` and the `VerticalAlignment` properties describe how to
+    HeightSpecification = 600
+};
+
+RelativeLayout.SetLeftRelativeOffset(yellowView, 0.5f);
+RelativeLayout.SetRightRelativeOffset(yellowView, 0.5f);
+RelativeLayout.SetTopRelativeOffset(yellowView, 0.5f);
+RelativeLayout.SetBottomRelativeOffset(yellowView, 0.5f);
+
+RelativeLayout.SetHorizontalAlignment(yellowView, RelativeLayout.Alignment.Center);
+RelativeLayout.SetVerticalAlignment(yellowView, RelativeLayout.Alignment.Center);
+relativeLayoutView.Add(yellowView);
+```
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    <div id="RelativeLayout-YellowView-Xaml" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+- `RelativeLayout.LeftRelativeOffset="0.5"` - view's left is on center of the parent's view.
+- `RelativeLayout.RightRelativeOffset="0.5"` - view's right is on center of the parent's view.
+- `RelativeLayout.TopRelativeOffset="0.5"` - view's top is on center of the parent's view.
+- `RelativeLayout.BottomRelativeOffset="0.5"` - view's bottom is on center of the parent's view.
+- `WidthSpecification` and `HeightSpecification` set the size of the view.
+- `RelativeLayout.HorizontalAlignment="Center"` and `RelativeLayout.VerticalAlignment="Center"` - position view on the center of the container.
+
+```xaml
+<View x:Name="yellowView"
+    BackgroundColor="Yellow"
+    WidthSpecification="600"
+    HeightSpecification="600"
+    RelativeLayout.LeftRelativeOffset="0.5"
+    RelativeLayout.RightRelativeOffset="0.5"
+    RelativeLayout.TopRelativeOffset="0.5"
+    RelativeLayout.BottomRelativeOffset="0.5"
+    RelativeLayout.HorizontalAlignment="Center"
+    RelativeLayout.VerticalAlignment="Center"/>
+```
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
+
+### Green view
+
+The following code shows how to create the green view to relate to yellowView for every target:
+![RelativeLayoutExampleGreenView](./media/relative_layout_example_green_view.png)
+
+<div id="TabSection5">
+    <div class="sampletab " id="ProjectCreateTab">
+        <button id="RelativeLayout-GreenView-CSharp" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-GreenView-CSharp', 'TabSection5') ">C#</button>
+        <button id="RelativeLayout-GreenView-Xaml" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-GreenView-Xaml', 'TabSection5') ">Xaml</button>
+    </div>
+    <div id="RelativeLayout-GreenView-CSharp" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+- View's targets are set to the `yellowView`.
+- `RelativeLayout.SetLeftRelativeOffset(greenView, 0.0f)` - view's left is at the yellowView's left.
+- `RelativeLayout.SetRightRelativeOffset(greenView, 0.2f)` - view's right is at the position of 0.2 width from the yellowView's left.
+- `RelativeLayout.SetTopRelative Offset(greenView, 0.0f)` - view's top is at the yellowView's top.
```

Comment:
Would you please check if the space in the middle is needed?

```suggestion
- `RelativeLayout.SetTopRelativeOffset(greenView, 0.0f)` - view's top is at the yellowView's top.
```

Reply:
Fixed